Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -152,6 +155,19 @@ int RunVertices(int vtx_attr_group, int primitive, int count, DataReader src, bo
 	if (skip_drawing || is_preprocess)
 		return size;
 
+	// Hide Objects Code code
+	if (m_LocalCoreStartupParameter.num_object_removal_codes)
+	{
+		for (const SkipEntry& entry : m_LocalCoreStartupParameter.object_removal_codes)
+		{
+			if (!memcmp(src.GetPointer(), entry.data(), entry.size()))
```

Comment:
Was just testing it out.  On the upside, it does uniquify the objects quite nicely, but on the downside it changes the amount of instructions from about 12 per code to about 250, which causes some framerate drop in extreme situations.  One of my stress tests is the opening scene from Metroid Prime with the culling turned off (we run it like that for VR), with the 16:9 bars removed, and in that scene the hashing caused some frame drops, while the current code is completely smooth.

Maybe in the future there could be an option to enable a hashed check if the object required it?  You could choose a hashed code or a non hashed code, and not do any hashing unless a code requested it?